### DISCUSSION TITLE
Hide "iss" in Token

### DISCUSSION THREAD

**Discussion by mm-btl** [keycloak/keycloak#25160] [Q&A]
I have as result of a Penetration Test: "The JWT containt the internal hostname of keycloak".
This means, for example: "iss": "http://ab-keycloak/realms...."
Is there a possibilty to hide this in the token?

**Comment by bonnm** [comment]
According to the OIDC specification (https://openid.net/specs/openid-connect-core-1_0.html#IDToken) the iss claim is required. It is needed by clients to verify the validity of the token.
